src/features/staking/utils/convertUnixToDays.ts

```diff
@@ -0,0 +1,35 @@
+export const convertUnixToDays = (maturity: bigint | undefined) => {
```

Comment:
I tend to use `date-fns` for this, but DIY is also fine :)

Reply:
At this point I'll keep the DIY solution since it's already there